packages/alea-frontend/pages/api/login.ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { executeAndEndSet500OnError } from './comment-utils';
+import bcrypt from 'bcrypt';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const { email, password } = req.body;
+  const existingUser = (await executeAndEndSet500OnError(
+    `SELECT * FROM userInfo WHERE userId = ?`,
```

Comment:
avoid selecting `*`. If you only need password just select that. Otherwise there will be unnecessary data transfer from database to server